Arm/rover_arm_system.hpp

```diff
@@ -121,129 +121,100 @@ class RoverArmSystem : public sjsu::common::RoverSystem
 
   void HandleArmMovement()
   {
-    // rotunda
-    units::angular_velocity::revolutions_per_minute_t rotunda_speed(
-        static_cast<float>(mc_data.rotunda_speed));
-    units::angle::degree_t rotunda_angle(
-        static_cast<float>(mc_data.rotunda_angle));
-    // shoulder
-    units::angular_velocity::revolutions_per_minute_t shoulder_speed(
-        static_cast<float>(mc_data.shoulder_speed));
-    units::angle::degree_t shoulder_angle(
-        static_cast<float>(mc_data.shoulder_angle));
-    // elbow
-    units::angular_velocity::revolutions_per_minute_t elbow_speed(
-        static_cast<float>(mc_data.elbow_speed));
-    units::angle::degree_t elbow_angle(static_cast<float>(mc_data.elbow_angle));
-    // wrist
-    units::angular_velocity::revolutions_per_minute_t wrist_speed(
-        static_cast<float>(mc_data.wrist_speed));
-    // units::angle::degree_t
-    // wrist_roll(static_cast<float>(mc_data.wrist_roll));
-    // units::angle::degree_t
-    // wrist_pitch(static_cast<float>(mc_data.wrist_pitch));
-
     // TODO: implement different arm drive modes in this function.
 
     // D drive mode
-    MoveRotunda(rotunda_speed, rotunda_angle);
-    MoveShoulder(shoulder_speed, shoulder_angle);
-    MoveElbow(elbow_speed, elbow_angle);
+    MoveRotunda(mc_data_.rotunda_speed, mc_data_.rotunda_angle);
+    MoveShoulder(mc_data_.shoulder_speed, mc_data_.shoulder_angle);
+    MoveElbow(mc_data_.elbow_speed, mc_data_.elbow_angle);
   }
 
-  void MoveWrist(){};
+  void MoveWrist()
+  {
+
+  }
 
   void HomeAll()
   {
-    // first getting the accelerometer readings/storing readings
-    Accelerometer::Acceleration_t rotunda_acceleration =
+    // first reading the accelerometer data
+    Joint::Acceleration rotunda_acceleration =
         rotunda_.GetAccelerometerData();
-    Accelerometer::Acceleration_t shoulder_acceleration =
+    Joint::Acceleration shoulder_acceleration =
         shoulder_.GetAccelerometerData();
-    // then homing the different parts of the arm
     HomeShoulder(rotunda_acceleration, shoulder_acceleration);
 
-    Accelerometer::Acceleration_t elbow_acceleration =
+    Joint::Acceleration elbow_acceleration =
```

Comment:
Might be worth making these local member variables for holding acceleration data that way we could possibly break it up later or run some checks on them or something. Might be a bit silly though, who knows